docs/UserGuide.md

```diff
@@ -75,6 +75,14 @@ CONNECTIONS is a **desktop app for managing contacts, optimized for use via a Co
 
 </div>
 
+### Upcoming Birthdays
```

Comment:
Maybe you want to add this to the list of features on top? So that people can navigate here

Reply:
done